src/main/java/io/github/vampirestudios/raa/config/screen/dimensions/RAADimensionDescriptionListWidget.java

```diff
@@ -176,46 +176,46 @@ private static void openClothConfigForMaterial(DimensionListScreen og, Dimension
             );*/
             SubCategoryBuilder biomeData = eb.startSubCategory("config.title.raa.biomeData").setExpended(false);
             biomeData.add(
-                    eb.startStrField("config.field.raa.biomeData.id", material.getBiomeData().getId().getPath())
-                            .setDefaultValue(material.getBiomeData().getId().getPath())
-//                            .setSaveConsumer()
+                    eb.startStrField("config.field.raa.biomeData.id", material.getBiomeData().getId().toString())
```

Comment:
change this back to getPath since we don't want people to be able to edit the modid for the biomes